diff --git a/node_modules/@base-ui/react/esm/floating-ui-react/middleware/arrow.js b/node_modules/@base-ui/react/esm/floating-ui-react/middleware/arrow.js
index 6d72afc..7730f32 100644
--- a/node_modules/@base-ui/react/esm/floating-ui-react/middleware/arrow.js
+++ b/node_modules/@base-ui/react/esm/floating-ui-react/middleware/arrow.js
@@ -3,9 +3,9 @@ import { clamp, evaluate, getAlignment, getAlignmentAxis, getAxisLength, getPadd
  * Fork of the original `arrow` middleware from Floating UI that allows
  * configuring the offset parent.
  */
-export const baseArrow = options => ({
+export const baseArrow = (options, optionsValue = options) => ({
   name: 'arrow',
-  options,
+  options: optionsValue,
   async fn(state) {
     const {
       x,
@@ -88,7 +88,4 @@ export const baseArrow = options => ({
  * This wraps the core `arrow` middleware to allow React refs as the element.
  * @see https://floating-ui.com/docs/arrow
  */
-export const arrow = (options, deps) => ({
-  ...baseArrow(options),
-  options: [options, deps]
-});
\ No newline at end of file
+export const arrow = (options, deps) => baseArrow(options, [options, deps]);
\ No newline at end of file
diff --git a/node_modules/@base-ui/react/floating-ui-react/middleware/arrow.js b/node_modules/@base-ui/react/floating-ui-react/middleware/arrow.js
index e228afa..9fb78b4 100644
--- a/node_modules/@base-ui/react/floating-ui-react/middleware/arrow.js
+++ b/node_modules/@base-ui/react/floating-ui-react/middleware/arrow.js
@@ -9,9 +9,9 @@ var _utils = require("@floating-ui/utils");
  * Fork of the original `arrow` middleware from Floating UI that allows
  * configuring the offset parent.
  */
-const baseArrow = options => ({
+const baseArrow = (options, optionsValue = options) => ({
   name: 'arrow',
-  options,
+  options: optionsValue,
   async fn(state) {
     const {
       x,
@@ -95,8 +95,5 @@ const baseArrow = options => ({
  * @see https://floating-ui.com/docs/arrow
  */
 exports.baseArrow = baseArrow;
-const arrow = (options, deps) => ({
-  ...baseArrow(options),
-  options: [options, deps]
-});
+const arrow = (options, deps) => baseArrow(options, [options, deps]);
 exports.arrow = arrow;
\ No newline at end of file
